src/components/onboarding/OnBoardingModal/OnBoardingContentContainer.tsx

```diff
@@ -61,14 +63,23 @@ export default function OnBoardingContentContainer({
     subtitle = 'Your profile is ready to use, and you can edit it at any time.'
   }
 
-  const showContinueBtn = !loading && !success && !hideSubmitBtn
+  const showContinueBtn = !loading && !success && !hideSubmitBtn && currentStep !== 'signer'
 
   return (
     <>
       {openState === 'full-on-boarding' && !success && (
         <div className={clsx('d-flex justify-content-between mb-3')}>
           <MutedDiv className={clsx('d-flex align-items-center')}>
-            <ArrowLeftOutlined className='mr-2' onClick={loading ? undefined : onBackClick} />
+            <ArrowLeftOutlined
+              className='mr-2'
+              onClick={
+                loading
+                  ? undefined
+                  : () => {
+                      onBackClick()
+                    }
+              }
```

Comment:
Looks unclear for me...

Maybe better `!loading && () => onBackClick()`

Reply:
I got `parse error` when using `&&`. 
Maybe we can try `!loading ? () => onBackClick() : undefined` ? No `parse error`

Reply:
Implemented the idea above in 1b4406b